docs/serving/using-a-custom-domain.md

```diff
@@ -18,76 +11,70 @@ To change the {default-domain} value there are a few steps involved:
-   ```
-
-1. Edit the file to replace `example.com` with the domain you'd like to use,
-   remove the `_example` key and save your changes.
-   In this example, we configure `mydomain.com` for all routes:
-
-   ```yaml
-   apiVersion: v1
-   data:
-     mydomain.com: ""
-   kind: ConfigMap
-   [...]
-   ```
+     ```bash
+     kubectl edit cm config-domain --namespace knative-serving
+     ```
+
+     This command opens your default text editor and allows you to edit the [config
+     map](https://github.com/knative/serving/blob/main/config/core/configmaps/domain.yaml).
+
+     ```yaml
+     apiVersion: v1
+     data:
+       _example: |
+         ################################
+         #                              #
+         #    EXAMPLE CONFIGURATION     #
+         #                              #
+         ################################
+         # ...
+         example.com: |
+     kind: ConfigMap
+     ```
+
+1. Edit the file to replace `example.com` with the domain you'd like to use, remove the `_example` key and save your changes. In this example, we configure `mydomain.com` for all routes:
+
+     ```yaml
+     apiVersion: v1
+     data:
+       mydomain.com: ""
+     kind: ConfigMap
+     [...]
+     ```
 
 ## Apply from a file
 
 You can also apply an updated domain configuration:
 
-1. Create a new file, `config-domain.yaml` and paste the following text,
-   replacing the `example.org` and `example.com` values with the new domain you
-   want to use:
-
-   ```yaml
-   apiVersion: v1
-   kind: ConfigMap
-   metadata:
-     name: config-domain
-     namespace: knative-serving
-   data:
-     # These are example settings of domain.
-     # example.org will be used for routes having app=prod.
-     example.org: |
-       selector:
-         app: prod
-     # Default value for domain, for routes that does not have app=prod labels.
-     # Although it will match all routes, it is the least-specific rule so it
-     # will only be used if no other domain matches.
-     example.com: ""
-   ```
+1. Create a new file, `config-domain.yaml` and paste the following text, replacing the `example.org` and `example.com` values with the new domain you want to use:
+
+     ```yaml
+     apiVersion: v1
+     kind: ConfigMap
+     metadata:
+       name: config-domain
+       namespace: knative-serving
+     data:
+       # These are example settings of domain.
+       # example.org will be used for routes having app=prod.
+       example.org: |
+         selector:
+           app: prod
+       # Default value for domain, for routes that does not have app=prod labels.
+       # Although it will match all routes, it is the least-specific rule so it
+       # will only be used if no other domain matches.
+       example.com: ""
+     ```
 
 1. Apply updated domain configuration to your cluster:
 
-   ```bash
-   kubectl apply --filename config-domain.yaml
-   ```
+     ```bash
+     kubectl apply --filename config-domain.yaml
+     ```
```

Comment:
these apply commands can be removed if you're following the style guide for the kubectl edit commands